components/Form/index.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+
+export default function FormComponent({
+  onSubmitEvent = () => {}
+}) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+  
+    onSubmitEvent({
+      destination: data.destination,
+      from: data.from,
+      to: data.to,
+      entryTitle: data.entryTitle,
+      entry: data.entry,
+    });
+    event.target.reset();
+  }
+  return ( 
+<>
+  <StyledForm   onSubmit={(event) => handleSubmit(event)}>
```

Comment:
```suggestion
  <StyledForm   onSubmit={handleSubmit}>
```
Since you are passing the event straight to your handleSubmit and nothing more, you could write it as this instead and let the onClick call your handleSubmit directly.

Reply:
changed it